dgraph/Makefile

```diff
@@ -18,7 +18,14 @@ BIN            = dgraph
 BUILD         ?= $(shell git rev-parse --short HEAD)
 BUILD_DATE    ?= $(shell git log -1 --format=%ci)
 BUILD_BRANCH  ?= $(shell git rev-parse --abbrev-ref HEAD)
+
+# use a large version number v99 on master branch
+ifeq ($(BUILD_BRANCH),master)
+BUILD_VERSION ?= 'v99'
```

Comment:
I presume you will also have the latest git commit-id in the version string.

Reply:
No. Why do we need it? Sentry already has a "commit" field corresponding to each log which keeps the git commit id.